package.json

```diff
@@ -21,14 +21,13 @@
     "chai": "^4.2.0",
     "coveralls": "^3.0.6",
     "jshint": "^2.10.2",
-    "mocha": "^6.2.0",
-    "nyc": "^14.1.1",
-    "q-io": "^1.13.6"
+    "mocha": "^8.3.1",
```

Comment:
Updated mocha to pick up a newer version of lodash for [this advisory](https://npmjs.com/advisories/1523).